bucket/openoffice.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "4.1.7",
+    "description": "Office software suite for word processing, spreadsheets, presentations, graphics, databases and more",
+    "homepage": "https://www.openoffice.org",
+    "license": "Apache-2.0",
+    "url": "https://archive.apache.org/dist/openoffice/4.1.7/binaries/en-GB/Apache_OpenOffice_4.1.7_Win_x86_install_en-GB.exe#/dl.7z",
+    "hash": "sha512:f245545b5c3d434dc0407aee4e807dad5d552dcb156b0367d5314ae83a2257180e8f1ef28d76116cf90582767093e2180063ea0fcd20ce72541337826b145850",
+    "installer": {
+        "script": [
+            "(Get-ChildItem \"$dir\" 'openoffic*.msi').FullName | Expand-MsiArchive -DestinationPath \"$dir\\App\"",
+            "Get-ChildItem \"$dir\" -Exclude 'App' | Remove-Item -Recurse",
+            "Move-Item \"$dir\\App\\*\" \"$dir\" -Force",
+            "Remove-Item \"$dir\\App\" -Recurse"
```

Comment:
Force and Recurse is not needed. Or just use `movedir \"$dir\\App\" \"$dir\"` ? I don't know if we should use it out of scoop core.